Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -10,7 +10,6 @@
 ly_get_list_relative_pal_filename(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME})
 ly_get_list_relative_pal_filename(common_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/Common)
 
-set(LY_STATISTICAL_PROFILING_ENABLED OFF CACHE BOOL "Enables statistical profiling when using AZ_PROFILE_SCOPE. If True, it takes effect only if RAD Telemetry is disabled.")
```

Comment:
I know @galibzon added the Statistical profiling support about a year ago.
I imagine there is still a use for it.

Reply:
@galibzon can you weigh in on whether this will be needed given the current work on the Atom CPU profiler? The immediate step after this PR is in will be to move the Atom CPU profiler into core (separating it from the ImGUI portion which will still be used to visualize the results). If this is still needed I can do the work needed to bring it back before merging in this change

Reply:
The Statistical profiler is useful to collect running statistics. In other words, it saves a lot of memory because it doesn't keep track of the history of the collected data. It measures Min, Max, Avg, Stdev, et al without storing data.

Reply:
The idea is that I can collect Running statistics by leveraging the same AZ_PROFILE markers.

Reply:
OK if this is functionality not provided in the Atom CPU profiler today, would it be better to roll this functionality into that? Or would you prefer to keep this as-is